### PR TITLE
feature(calendar): add maxDate

### PR DESCRIPTION
**Pull request by cwklausing**
* Updated calendar to include a maxDate param. 
* Fixes a previously failing test by fixing the date format.

### CODE REVIEW

**Comment by lfender6445** [comment]
@cwklausing is there a way i can see this in action without having to run it myself? 

im wondering if you can point to it from an existing review env

File: src/Calendar.js [hunk 1]
```diff
@@ -31,8 +32,9 @@ export default class Calendar extends Component {
 
   handleDateSelected(evt) {
     const dateBefore = !isBefore(evt.target.dataset.date, new Date(format(this.props.minDate, 'MM/DD/YYYY')))
+    const dateAfter = !isAfter(evt.target.dataset.date, new Date(format(this.props.maxDate, 'MM/DD/YYYY')))
 
-    if (dateBefore) {
+    if (dateBefore && dateAfter) {
```
Comment (lfender6445) [inline src/Calendar.js]: this is supposed to be like 'not before this date and not after' right? 

if so could we update verbiage?

Comment (tadjohnston) [inline src/Calendar.js]: I think this is wrong.  I would think the following scenarios maybe:
```
dateBefore -> set date to evt.target.dataset.date
dateAfter -> set date to dateAfter - 1
```

Either way your conditional should at the very least be:
```
if (dateBefore || dateAfter) {
```

Comment (orionstein) [reply src/Calendar.js]: Yeh, it can't be both before and after

Comment (cwklausing) [reply src/Calendar.js]: Yeah, agreed. I'll update the verbiage.

**Pull request merged**